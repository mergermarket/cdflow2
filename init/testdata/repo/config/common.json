{
  "env": "%{env}",
  "location": "%{boilerplate_location}",
  "percent": "%{PERCENT}{keep}"
}
