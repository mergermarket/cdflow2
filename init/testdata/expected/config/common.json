{
  "env": "test",
  "location": "testdata/repo",
  "percent": "%{keep}"
}
